languages/js/src/dataFiltering.test.ts

```diff
@@ -1098,5 +1098,5 @@ has_relation(repo: Repo, "parent", _: Issue{repo: repo});
     await oso.checkAuthz(gwen, 'create_issues', Repo, []);
     await oso.checkAuthz(leina, 'create_issues', Repo, [pol]);
     await oso.checkAuthz(gabe, 'create_issues', Repo, []);
-  });
+  }, 60000);
```

Comment:
What's this doing?

Reply:
33898c05fc14dfa3761baa3dc861c6ebdb81af69

Reply:
So extending the test timeout?

Reply:
Yeah sorry for the terseness — I highlighted you [on Notion](https://www.notion.so/CI-flakes-d0b32be2bc2b40f09c6f7c56320ab8a3?d=ab20cbe9db694ae4b954d369e5ede383#9ab4c38d2f6041018e6720582497d205)